projects/cashmere/src/lib/pipes/age-pipe/age.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Date } from 'sugar';
```

Comment:
This would require a peerDependency to be added to Cashmere (currently the DatePicker doesn't strictly require Sugar). I'm working through how to open things up so you can depend on other libraries without requiring additional peer dependencies to the Cashmere library itself.

Reply:
Also, I think we would prefer to not shadow the global `Date`; IMHO it would be better to either:

* create an alias: `import { Date as SugarDate }` from 'sugar';
* import the functions directly: `import createDate from 'sugar/date/internal/createDate';`

Reply:
Fixed